README.md

```diff
@@ -414,6 +420,22 @@ bibutils.formats.extension = {
 };
 ```
 
+## Change Binary Path
+
+If you want to use a different set of BibUtils binaries, you can set the path to them
+using the exported function 'setBinaryPath'.
+
+```javascript
+bibutils.setBinaryPath('/usr/bin');
+```
+
+By default, using a custom path will not use a binary extension. If you need to set it
```

Comment:
I understand that this means a user just needs a file like "bib2xml", and not "bib2xml-linux", but I think the wording is unclear to a user.

Maybe something like this?

 ```suggestion
By default, using a custom path will not use a binary extension. 
(ie: bibutils.js will look for files like `bib2xml`, not `bib2xml-linux` or other variants)  
If you need to set it
```